src/Entity/Rating.php

```diff
@@ -13,45 +14,44 @@ class Rating
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @var int
      */
     private $id;
 
     /**
      * @ORM\Column(type="text")
+     * @var string
      */
     private $text;
 
     /**
      * @ORM\Column(type="smallint")
+     * @var int
      */
     private $value;
 
-    public function getId(): ?int
+    public function __construct(string $text, int $value)
     {
-        return $this->id;
-    }
+        if (Assertion::between($value, 1,5)) {
```

Comment:
nene bro, stačí zavolať ten `Assertion::between($value, 1,5)`, žiaden `if`, preklikni si to jak je to naimplementované, to samo vyhadzuje výnimku

Reply:
sakra , to mi tam zustalo, chtel jsem vyhazovat vyjimku